_config/site.yml

```diff
@@ -85,7 +85,21 @@ profiles:
       :cdn_http_base: *dev_cdn_http_base
       :cdn_out_dir: *dev_cdn_out_dir
       :cdn_version: *dev_cdn_version
-  
+
+  # Pull Request configuration
+  staging_docker_pr:
+    <<: *profile
+    dcp_base_url: http://searchisko:8080/
+      #TODO We should be able to use HOSTNAME
+    dcp_base_protocol_relative_url: //10.3.13.63:<%= ENV['SEARCHISKO_HOST_PORT'] %>/
```

Comment:
This won't work, we can't hard code IPs here because they may not always be the same. Isn't there a SEARCHISKO_HOST_IP or similar environment variable we can use here?